ea_init_coregctpopup.m

```diff
@@ -1,4 +1,19 @@
 function ea_init_coregctpopup(handles,options,handlestring)
+% 
```

Comment:
Please provide a description of the function

Reply:
```
% Initialize CT coregistration methods popupmenu.
%
% INPUTS:
%    handles: LEAD GUI handle
%    refine: deprecated.
```